avalanche/logging/wandb_logger.py

```diff
@@ -49,6 +55,11 @@ def import_wandb(self):
                 'Please run "pip install wandb" to install wandb')
         self.wandb = wandb
 
+    def args_parse(self):
+        self.init_kwargs = {"project": self.project_name, "name": self.run_name}
+        if self.params:
+            self.init_kwargs |= self.params
```

Comment:
Maybe here you want to merge the two dictionaries? I don't think the `|=` operation is valid on two dicts.

Reply:
Oh right, fixed the merge. My bad